routes/users.go

```diff
@@ -34,6 +35,20 @@ type registerRequest struct {
 	DateOfBirth *time.Time `json:"dateOfBirth"`
 }
 
+type updateUserRequest struct {
+	PasswordHash    []byte
```

Comment:
If we want to be able to update password, they need to send the new password in plain text and we'll update the password hash in the backend, but this would complicate the access token since we can't revoke it

Reply:
removed field